### ISSUE TITLE
Add task for fetching logs

### ISSUE THREAD

**Issue by nicorikken** [CISOfy/lynis-ansible#7]
Ease fetching the resulting logs and report data, as the terminal output will only go so far.

**Comment by mboelen** [comment]
Is this just a matter of copying back the data? Data is in /var/log/lynis.log and /var/log/lynis-report.dat by default. Can be changed with flags.

**Comment by nicorikken** [comment]
Indeed, but I'd like to have a default task provided for this common use-case. Otherwise many people will be writing the same lines of code.
